tests/e2e/v3_curl_kv_test.go

```diff
@@ -118,18 +153,30 @@ func testV3CurlTxn(cx ctlCtx) {
 	}
 	m := &runtime.JSONPb{}
 	jsonDat, jerr := m.Marshal(txn)
-	if jerr != nil {
-		cx.t.Fatal(jerr)
-	}
+	require.NoError(cx.t, jerr)
+
 	expected := `"succeeded":true,"responses":[{"response_put":{"header":{"revision":"2"}}}]`
-	p := cx.apiPrefix
-	if err := e2e.CURLPost(cx.epc, e2e.CURLReq{Endpoint: path.Join(p, "/kv/txn"), Value: string(jsonDat), Expected: expect.ExpectedResponse{Value: expected}}); err != nil {
-		cx.t.Fatalf("failed testV3CurlTxn txn with curl using prefix (%s) (%v)", p, err)
-	}
+	err := e2e.CURLPost(cx.epc, e2e.CURLReq{Endpoint: "/v3/kv/txn", Value: string(jsonDat), Expected: expect.ExpectedResponse{Value: expected}})
```

Comment:
it could be easy to read if using multiple lines here.

Reply:
updated.